hono/src/services/items.ts

```diff
@@ -104,7 +108,7 @@ app.delete(
         }
 
         // Delete the specified item
-        return tx.delete(items).where(eq(items.id, params.id)).returning()
```

Comment:
Since you add `{ onDelete: 'cascade' }` to the schema, outfits in itemsToOutfits might be automatically deleted after deleting items. So delete outfits might not be necessary (line 101). You can try verifying this behavior locally.

Reply:
This functionality should actually work as expected and deleting the outfit itself is required. The `onDelete: 'cascade'` action is implemented as such: item is deleted -> all entries in items_to_outfit table matching item id is deleted OR outfit is deleted -> all entries in items_to_outfits table matching outfit id is deleted. If an outfit/item is deleted, it doesn't cascade all the way over to the related table.

However, you do bring up a good point where the schema *should* automatically cascade this items deletion over to all related outfits. I'll look into it.

Reply:
I gave it a look and it appears my implementation is the only reasonable way to achieve the behavior that we want. Adding an additional cascade on delete from a many to many relationship is not possible unless you use a separate "trigger". In other words, this functionality cannot be implemented in the schema easily. Therefore, I'd argue that this implementation is sound for now.

Reply:
Ok sounds good, we can roll with your approach then.